lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -378,7 +378,7 @@ class ImportCommand < HammerCLIForeman::Command
       command_name "import"
 
       success_message _("Content view imported.")
-      failure_message _("Could not import the content view.")
+      failure_message _("Could not import the content view")
```

Comment:
Why drop the period?

Reply:
Removed the period on line 381 because it made the error message look odd:

`before: Could not import the content view.:`

`after: Could not import the content view:`

Plus it looks like the other failures do not have them but the success do:

https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/content_view_version.rb#L74

https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/content_view_version.rb#L87

https://github.com/Katello/hammer-cli-katello/blob/master/lib/hammer_cli_katello/content_view_version.rb#L135